go-crud-psql/db/connect.go

```diff
@@ -0,0 +1,106 @@
+package db
+
+import (
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/jinzhu/gorm"
+	_ "github.com/jinzhu/gorm/dialects/postgres"
+)
+
+type Store struct {
+	DB *gorm.DB
+}
+
+func Init() *Store {
```

Comment:
Would be more idiomatic.
```suggestion
func NewStore() *Store {
```